datarepo-clienttests/src/main/java/testscripts/KubeConfig.java

```diff
@@ -0,0 +1,177 @@
+      if (bulkLoadArrayJobResponse.getJobStatus().equals(JobModel.JobStatusEnum.RUNNING)) {
+        System.out.println("Scaling pods back up to 4");
+        KubernetesClientUtils.modifyKubernetesPostDeployment(4);
+      }
+    }
+    /*---------------------------------------------*/
+
+    // wait for the job to complete and print out results
+    getAndDisplayResults(repositoryApi, bulkLoadArrayJobResponse);
+  }
+
+  // This should run about 5 minutes on 2 DRmanager instances. The speed of loads is when
+  // they are not local is about 2.5GB/minutes. With a fixed size of 1GB, each instance should do
+  // 2.5 files per minute, so two instances should do 5 files per minute. To run 5 minutes we should
+  // run 25 files.
+  private BulkLoadArrayRequestModel buildBulkLoadFileRequest() {
+    String loadTag = FileUtils.randomizeName("longtest");
+
+    BulkLoadArrayRequestModel arrayLoad =
+        new BulkLoadArrayRequestModel()
+            .profileId(billingProfileModel.getId())
+            .loadTag(loadTag)
+            .maxFailedFileLoads(filesToLoad); // do not stop if there is a failure.
+
+    System.out.println(
+        "longFileLoadTest loading "
+            + filesToLoad
+            + " files into dataset id "
+            + datasetSummaryModel.getId());
+
+    // There are currently 26 source files, so if ingesting more files: continue to loop through the
+    // source files,
+    // but point to new target files file paths
+    int numberOfSourceFiles = 26;
+    for (int i = 0; i < filesToLoad; i++) {
+      String fileBasePath = "/fileloadscaletest/file1GB-%02d.txt";
+      String sourcePath =
+          "gs://jade-testdata-uswestregion" + String.format(fileBasePath, i % numberOfSourceFiles);
+      String targetPath = "/" + loadTag + String.format(fileBasePath, i);
+      System.out.println("Source Path: " + sourcePath + "; Target Path: " + targetPath);
+
+      BulkLoadFileModel model = new BulkLoadFileModel().mimeType("application/binary");
+      model.description("bulk load file " + i).sourcePath(sourcePath).targetPath(targetPath);
+      arrayLoad.addLoadArrayItem(model);
+    }
+
+    return arrayLoad;
+  }
+
+  private void getAndDisplayResults(RepositoryApi repositoryApi, JobModel bulkLoadArrayJobResponse)
+      throws Exception {
+    bulkLoadArrayJobResponse =
+        DataRepoUtils.waitForJobToFinish(repositoryApi, bulkLoadArrayJobResponse);
+    BulkLoadArrayResultModel result =
+        DataRepoUtils.expectJobSuccess(
+            repositoryApi, bulkLoadArrayJobResponse, BulkLoadArrayResultModel.class);
+
+    BulkLoadResultModel loadSummary = result.getLoadSummary();
+    System.out.println("Total files    : " + loadSummary.getTotalFiles());
+    System.out.println("Succeeded files: " + loadSummary.getSucceededFiles());
+    System.out.println("Failed files   : " + loadSummary.getFailedFiles());
+    System.out.println("Not Tried files: " + loadSummary.getNotTriedFiles());
```

Comment:
Out of scope for this PR, but I think this is a great example of how our result reporting will need to be different for performance/resilience vs functional tests. It would be great to have the TestRunner report this information back as it dumps the UserJourneyResults out at the end (maybe we can allow the userjourney method to return any sub-class of a UserJourneyResult?). Anyway, I'd appreciate your thoughts on this. Let's discuss outside fo this PR and then make new ticket.